core/divelist.c

```diff
@@ -25,6 +25,7 @@
  * void mark_divelist_changed(int changed)
  * int unsaved_changes()
  * void remove_autogen_trips()
+ * void sort_table(struct dive_table *table)
```

Comment:
BTW: this is from the very, very earliest days of development. Only a couple of files have these lists of functions at the beginning, and I bet they are ridiculously out of date. We should just remove them.